### ISSUE TITLE
 Predicted boxes or scores contain Inf/NaN. Training has diverged

### ISSUE THREAD

**Issue by 180041123-Atiq** [XXX-YES/FSOD#5]
It is really appreciated that you open sourced your code for your paper. I followed the setup instructions and attempted to run on UODD dataset only, but I have came across this issue which was previously closed. I used R50-FPN with 3x, pretrained on MS COCO. I am adding the 5_shot.log file link [here](https://pastebin.pl/view/7464f00e).

**Comment by 180041123-Atiq** [comment]
I solved it with this equation,
num_gpu = 1
bs = (num_gpu * 2)
cfg.SOLVER.BASE_LR = 0.02 * bs / 16  # pick a good LR
since I have 1 GPU batch size 2, I decreased the learning rate accordingly.
Somehow I got 14.815 mAP higher than stated in the paper.
Again thank you for your wonderful work.
